helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedRebalancer.java

```diff
@@ -443,7 +443,7 @@ private void refreshBaseline(ResourceControllerDataProvider clusterData,
     Map<String, ResourceAssignment> newAssignment =
         optimalAssignment.getOptimalResourceAssignment();
 
-    LOG.info("Finish calculating. Time spent: {}ms.", System.currentTimeMillis() - startTime);
+    LOG.info("Finish calculateAssignment(). Took: {} ms.", System.currentTimeMillis() - startTime);
```

Comment:
"Finish calculating assignment."
Usually, we don't like pasting code details in the log. Consider what if we change the method name in the future?